c/src/ml-api-service-agent-client.c

```diff
@@ -31,12 +31,24 @@ typedef struct
 static MachinelearningServicePipeline *
 _get_proxy_new_for_bus_sync (void)
 {
+  MachinelearningServicePipeline *mlsp;
+
   /** @todo deal with GError */
-  return
-      machinelearning_service_pipeline_proxy_new_for_bus_sync
+  mlsp = machinelearning_service_pipeline_proxy_new_for_bus_sync
+      (G_BUS_TYPE_SYSTEM, G_DBUS_PROXY_FLAGS_NONE,
```

Comment:
Q: is it possible to check bus-type? (availability of G_BUS_TYPE_SYSTEM or G_BUS_TYPE_SESSION)

Reply:
Well, I think checking the return value of this function seems proper to do that.

Reply:
In my opinion, this patch is OK as `Workaround`.
The reason why try both `G_BUS_TYPE_SYSTEM` and `G_BUS_TYPE_SESSION` is that our CI test uses `Session Bus` but Tizen devices use `System Bus`. In this case, we build two libraries: One is for Device, the other is for CI Test.

Anyway, to submit the platform image, it's ok.